### PR TITLE
Created a top level playbooks dir and scripts dir

### PR DESCRIPTION
**Pull request by ingvagabund**
Polish https://github.com/kubernetes/contrib/pull/967 work done by Steve Louie. Thanks for your effort starting the effort.

### CODE REVIEW

File: ansible/README.md [hunk 1]
```diff
@@ -31,20 +31,19 @@ kube-minion-test-[1:2].example.com
 
 ### Configure Cluster options
 
-Look through all of the options in `group_vars/all.yml` and
+Look though all of the options in `inventory/group_vars/all.yml` and
```
Comment (sttts) [inline ansible/README.md]: through

**Comment by ingvagabund** [comment]
@redhat-k8s-bot test please

File: ansible/README.md [hunk 2]
```diff
@@ -14,7 +14,7 @@ can be real hardware, VMs, things in a public cloud, etc. Anything that you can
 
 ### Configure inventory
 
-Add the system information gathered above into the 'inventory' file, or create a new inventory file for the cluster.
+Add the system information gathered above into the 'inventory' file, or create a new inventory file for the cluster. Place the 'inventory' file into the `./inventory` directory.
```
Comment (sttts) [inline ansible/README.md]: compare my old comment in the other PR

File: ansible/README.md [hunk 3]
```diff
@@ -31,20 +31,19 @@ kube-minion-test-[1:2].example.com
...
 
-Look through all of the options in `group_vars/all.yml` and
+Look though all of the options in `inventory/group_vars/all.yml` and
 set the variables to reflect your needs. The options are described there
 in full detail.
 
 ## Running the playbook
 
-After going through the setup, run the setup script provided:
+After going through the setup, run the deploy-cluster script provided by going into the scripts directory:
```
Comment (sttts) [inline ansible/README.md]: .... from the scripts directory

File: ansible/README.md (hunk 2 above)
Comment (ingvagabund) [inline ansible/README.md]: updated

File: ansible/README.md (hunk 3 above)
Comment (ingvagabund) [inline ansible/README.md]: updated

**Comment by ingvagabund** [comment]
@redhat-k8s-bot test please

**Comment by sttts** [comment]
lgtm

**Comment by ingvagabund** [comment]
Just rebasing on top of the master. No change to commit.

**Pull request merged**